operating_procedures.md

```diff
@@ -62,33 +60,47 @@ In collaboration with Editors and Maintainers, Contributors help drive the evolu
 
+Editors are designated by the WG Chair and ratified by the Working Group Members.
+
 ### 2.2.3 Task Force Convener
 
 Task Force Conveners are dedicated Maintainers within the FinOps FOCUS project who take on the responsibility of leading specific technical workstreams or topics within Task Forces. This leadership role involves guiding the development and implementation of specialized areas of the project, ensuring that all activities align with the project's broader objectives. Conveners are essential in driving detailed technical work, fostering collaboration, and maintaining high standards within their respective domains.
 
 
 ### 2.2.4 FOCUS Group Maintainer
   
-  FG Maintainers are a subset of the FOCUS Group members who have been given write access to one or more FOCUS Working Group repositories within the FOCUS Project's Github organization. They will advance the day-to-day evolution of the specification and related work products.
+  FG Maintainers are a subset of the FOCUS Group (FG) members who have been given write access to one or more FOCUS Working Group repositories within the FOCUS Project's Github organization. They will advance the day-to-day evolution of the specification and related work products.
 
   FG Maintainers are responsible for ensuring that the contents of documents and work products accurately reflect the decisions that have been made by the group, and that the specification adheres to formatting and content guidelines. 
 
-  FG Chair(s) will designate a FG Maintainer or FG Maintainers for the FG. FG Maintainer(s) are overseen by of the FG Chair(s).
+  FG Chair(s) designates any FG Maintainer(s) for the FG. FG Maintainers are overseen by the FG Chair(s). Maintainer designations will be revisited at the discretion of the FG chair, typically at the conclusion of a release cycle.  
 
-  The FG Maintainer(s) MUST keep the project documentation up to date (e.g.: contributing, readme and release planning documents).
+  To be designated as a Maintainer, individuals MUST:
 
-  The FG Maintainer(s) MUST apply the "Review & Approval" process—covered in 6.1 of this document—to contributions submitted by the Working Group members.
+   * Be involved as a Contributor consistently over at least one release cycle
+   * Lead or significantly contribute technical content to a major Issue or PR 
+   * Regularly participate in the Review & Approval process, including meeting discussions, issue/PR comments, asynchronous discourse (e.g. Slack), and PR approvals
+   * Continue these activities through each release cycle
```

Comment:
i think it would be helpful to be clearer on when a maintainer will be "at risk" of being removed. This line is a little vague, it think something like "at the end of each release cycle, a review of maintainers contributions will be held and active maintainers will maintain their status going into the next release cycle" (obviously with better words). Maybe also something about being able to pause your status and resume if prior notice is provided (basically we understand life gets in the way, just don't drop off the face of the planet let us know you need to pause).....

Reply:
I think your concern is sufficiently handled in line 76, which includes:

> Maintainer designations will be revisited at the discretion of the FG chair, typically at the conclusion of a release cycle. 

I think that "discretion" handles for such cases recurring contribution and personal leave.  Let me know if you disagree.